.travis.yml

```diff
@@ -2,28 +2,27 @@ language: php
 
 sudo: false
 
-php: [5.3, 5.4, 5.5, 5.6, 7.0, hhvm]
+php: [5.4, 5.5, 5.6, 7.0, 7.1, hhvm]
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.3
+    - php: 7.1
       env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 5.6
+    - php: 5.4
```

Comment:
Not good. This would not test against a dev version of SF for instance, because they dropped support for 5.4. This job should run on the highest stable PHP version instead